docs/installationguide.md

```diff
@@ -178,6 +178,7 @@ The `config.http` section of the config file contains all the information needed
 transport protocol binding. The following options are accepted:
 
 * **port**: South Port where the HTTP listener will be listening for information from the devices.
+* **timeout**: HTTP Timeout for the http command endpoint (in miliseconds).
```

Comment:
I think that "http command endpoint" -> "HTTP endpoint" would be more accurate and aligned with doc style.

Reply:
Fixed in 4f68b31